### PR TITLE
chore: enable automated releases

### PR DESCRIPTION
**Pull request by merlinnot**
This Pull Request fulfills the following requirements:

- [x] The commit message follows our guidelines.
- [x] Tests for the changes have been added if needed.
- [x] Does not affect documentation or it has been added or updated.

### CODE REVIEW

File: .releaserc.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 branch: master
 ci: false
-dryRun: true
 debug: false
+dryRun: false
```
Comment (viestat) [inline .releaserc.yaml]: why do we need it with false? isn't this the default?

Comment (merlinnot) [reply .releaserc.yaml]: Ooops, too late. I thought it will be easier to switch it in the future if needed.

Comment (viestat) [reply .releaserc.yaml]: not big deal, was just curious

Comment (oleg-koval) [reply .releaserc.yaml]: we can keep such values turned to false, same for `debug` btw, so its easier to switch, as these two options will be most used during development/debugging

**Pull request merged**